index.js

```diff
@@ -138,15 +148,21 @@ class Ky {
 			this._options.prefixUrl += '/';
 		}
 
-		const url = new _globalThis.URL(this._options.prefixUrl + this._input, document.baseURI);
-		if (typeof searchParams === 'string' || searchParams instanceof _globalThis.URLSearchParams) {
-			url.search = searchParams;
-		} else if (searchParams && Object.values(searchParams).every(param => typeof param === 'number' || typeof param === 'string')) {
-			url.search = new _globalThis.URLSearchParams(searchParams).toString();
-		} else if (searchParams) {
-			throw new Error('The `searchParams` option must be either a string, `URLSearchParams` instance or an object with string and number values');
+		// Avoid using URL() if we can, as it does not support relative URLs
+		// and is yet another global to mock in a non-browser environment.
```

Comment:
This comment is confusing as right below it `URL` *is* actually used.

Reply:
True, I agree. Without the comment, though, I feel the purpose is not super obvious. Will try to think of a better way to describe or code it. Suggestions welcome!